README.md

```diff
@@ -137,14 +137,18 @@ user.getCards().then(new PromiseAction<List<Card>>() {
 ### Create new card
 
 ```java
+// You can create a simple card request with just the label and the currency.
 CardRequest cardRequest = new CardRequest("label", "USD");
 user.createCard(cardRequest);
+
+// Or a card request with the label, currency, current position and whether it is starred or not.
```

Comment:
`// Or a card request with the label, currency, position and whether it is starred or not.`